shell/shared/demo.js

```diff
@@ -162,3 +176,34 @@ Router.map(function () {
     }
   });
 });
+
+Router.map(function () {
+  this.route("appdemo", {
+    path: "/appdemo/:packageId",
+    waitOn: function () {
+      return Meteor.subscribe("packageInfo", this.params.packageId);
```

Comment:
Well, I noticed earlier that if I didn't do this, the app name would come up empty. I can re-test to find out if it's needed.